react/components/FilterOptionTemplate.js

```diff
@@ -18,6 +18,7 @@ import { useCssHandles } from 'vtex.css-handles'
 import styles from '../searchResult.css'
 import { SearchFilterBar } from './SearchFilterBar'
 import SettingsContext from './SettingsContext'
+import useOutsideClick from './../hooks/useOutsideClick'
```

Comment:
i think you forgot to commit this hook

Reply:
This hook already exists @thalytafabrine !